eng/run-test.sh

```diff
@@ -65,60 +65,12 @@ trap handle_ctrl_c INT
 ProjectRoot="$(dirname "$(dirname "$(realpath ${BASH_SOURCE[0]})")")"
 
 # Location parameters
-# OS/Configuration defaults
+# OS/ConfigurationGroup defaults
```

Comment:
Even though this is just a comment I would revert this back to `Configuration`. We don't use `ConfigurationGroup` anymore.